src/Core/CommandBus/Parser/CommandDefinition.php

```diff
@@ -29,7 +29,7 @@
 /**
  * Transfers commands and queries definition data
  *
- * @deprecated
+ * @deprecated Since 1.7.8.0, use CommandHandlerDefinition instead.
```

Comment:
Don't forget to trigger a deprecated: https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations